docs/interface_dtfj.md

```diff
@@ -47,6 +47,14 @@ DTFJ is implemented in pure Java and tools written using DTFJ can be cross-platf
 
 See the [DTFJ API documentation](api-dtfj.md). <!-- Link to API -->
 
+Note: If the code that loads DTFJ is in a module, the module must require the `openj9.dtfj` module. For example:
```

Comment:
The documentation is generic and this only applies to Java 11+ so pls mention that.

Reply:
Does it strictly apply to Java >= 11, or does it also apply to Java >= 9 with modules? I can test if needed.

Reply:
Perhaps making it explicit is helpful, but the code can't be in a module if you're not using Java 11+.

Reply:
It would apply to Java 9 and Java 10 if there were any supported VMs. Saying Java >= 9 might be technically more correct, but it might also give people the idea they should try finding jdk9, which isn't helpful. On reflection, I think it might be better without a reference to any version: the 'module' concept is the only real consideration here.